### PR TITLE
PR for #5: Cluster standard errors at the county level

### LINKED ISSUE #5: Cluster standard errors at the county-level
For [Practice Task 2: Sherlock Extension](https://github.com/gentzkow/lab-manual-archive/wiki/Practice-Task):

- Modify the code in analysis/code/analyze_data.py to cluster standard errors at the county-level
- Add a new regression table capturing the revised outputs to the paper in paper_slides/code/paper.lyx
- Caption the table mentioning the two-way fixed effects the model uses, and that the std. errors are clustered at the county level

### PR DESCRIPTION
**Pull request by linxicindyzeng**
Closes https://github.com/linxicindyzeng/template_archive/issues/5

Changes made:
- Modify code in [analyze_data.py](https://github.com/linxicindyzeng/template_archive/blob/5263895e3e14fd3e343bf2128856693ace16e426/analysis/code/analyze_data.py) to cluster standard errors at the county-level
- Make subsequent changes throughout the repository to add new regression tables with clustered standard errors, and caption the tables mentioning the two-way fixed effects, and that the std. errors are clustered at the county level
- Execute run_all.py on Sherlock (job 52539941)

Deliverables:
- Updated [paper](https://github.com/linxicindyzeng/template_archive/blob/5263895e3e14fd3e343bf2128856693ace16e426/paper_slides/output/paper.pdf)

Draftable Difference:
- See [here](https://draftable.com/compare/ZHGMzsxhqDVD)

@ShiqiYang2022 Thanks for reviewing!

### CODE REVIEW

File: analysis/log/make.log [hunk 1]
```diff
@@ -1,23 +1,27 @@
 --------------------------------------------------------------------------------
-Makelog started: 2024-09-03 14:59:20
+Makelog started: 2024-09-03 18:52:21
 Working directory: /Users/zlxcindy/template_archive/analysis
```
Comment (ShiqiYang2022) [inline analysis/log/make.log]: From the log file, it seems like you are pushing from your local end, not Sherlock?

**Review by ShiqiYang2022 — CHANGES_REQUESTED** [review]
@linxicindyzeng Thanks for the work here!

I have one comment left below. Others all looks great to me.

File: analysis/log/make.log (hunk 1 above)
Comment (linxicindyzeng) [reply analysis/log/make.log]: Thanks for pointing this out, I fixed it. Please check the [make.log](https://github.com/linxicindyzeng/template_archive/blob/5263895e3e14fd3e343bf2128856693ace16e426/analysis/log/make.log) file now. @ShiqiYang2022

Comment (linxicindyzeng) [reply analysis/log/make.log]: I also updated the PR detail with new links.

**Review by ShiqiYang2022 — APPROVED** [review]
I think the PR is now good to go, thanks @linxicindyzeng!

After you wrap up this issue, I will close the your intake task and write an email to Matt confirming your finish of onboarding.

**Comment by ShiqiYang2022** [comment]
(by the way, next time please remember to include your commit message with the #[issue number])

**Comment by linxicindyzeng** [comment]
> (by the way, next time please remember to include your commit message with the #[issue number])

Oh yes, I will remember next time! Thanks for the reminder.

**Pull request merged**